js/util-galery.js

```diff
@@ -0,0 +1,68 @@
+
+const bigPhoto = document.querySelector('.big-picture');
+const body = document.querySelector('.body');
```

Comment:
Если искать body, то нужно селектор поправить вот так
```suggestion
const body = document.querySelector('body');
```
Но, это не нужно, потому что body один в документе и ссылка на него всегда есть в нем, т.е. `document.body` и можно к нему просто обращаться где нужно чего него, т.е. `document.body.addEventListner(...)`